setup.py

```diff
@@ -62,22 +62,22 @@ class CleanCommand(Command):
     description = "Tidy up the project root"
     user_options: List[str] = []
 
-    def initialize_options(self):
+    def initialize_options(self) -> None:
         """Set default values for options."""
 
-    def finalize_options(self):
+    def finalize_options(self) -> None:
         """Set final values for options."""
 
     @staticmethod
-    def rm_all_files(files: List[str]):
+    def rm_all_files(files: List[str]) -> None:
         """Remove all files from the list"""
         for file in files:
             try:
                 os.remove(file)
             except Exception as e:  # noqa pylint: disable=broad-except
-                logger.warning("Error when removing %s: %s", file, e)
+                logger.warning(f"Error when removing {file}: {e}")
```

Comment:
Using F-strings for logger is bad-practice, please revert that. Read for example here why: 

https://www.siegescape.com/blog/2019/10/3/be-careful-with-your-f-strings